ROADMAP.md

```diff
@@ -14,30 +14,26 @@ The following features are considered for the near future:
 
 * **Windows support improvements**
 Antrea [supports Windows K8s Node](docs/windows.md) since version 0.7.0.
-However, a few features like network flow export are not supported for Windows
-Node yet. We will continue to add more features for Windows, and improve Antrea
-Agent and OVS installation on Windows Nodes.
+However, a few features including: Egress, NodePortLocal, IPsec encryption are
+not supported for Windows Node yet. We will continue to add more features for
+Windows, and improve Antrea Agent and OVS installation on Windows Nodes.
 
-* **IPv6 Pod network**
-Support IPv6 and IPv4/IPv6 dual-stack for Pod network. Right now Antrea supports
-only IPv4.
-
-* **Antrea NetworkPolicy**
-Antrea has started adding support for [Antrea native NetworkPolicy](docs/antrea-network-policy.md)
+* **Antrea NetworkPolicy enhancements**
+Antrea added support for [Antrea native NetworkPolicy](docs/antrea-network-policy.md)
```

Comment:
The policy doc refers to them as "Antrea-native policies"

same for new bullet points at the end (Node security, L7)

Reply:
I changed this one to Antrea-native policies, but for other references I feel "Antrea-native NetworkPolicies" is clearer.